src/raygui.h

```diff
@@ -2367,6 +2381,13 @@ int GuiTextBox(Rectangle bounds, char *text, int bufferSize, bool editMode)
     // Blink-cursor frame counter
     //if (!autoCursorMode) blinkCursorFrameCounter++;
     //else blinkCursorFrameCounter = 0;
+    if (blinkSettings.blinkEnabled) {
+        if (blinkCursorFrameCounter < (blinkSettings.framesShown + blinkSettings.framesHidden)) {
```

Comment:
this will probably cause issues, since multiple elements have their own blinkSettings, but the `blinkCursorFrameCounter` is global